### DISCUSSION TITLE
Allow removing of a user from your feed

### DISCUSSION THREAD

**Discussion by Nolram12345** [community/community#31863] [Other Feature Feedback, Questions, & Ideas]
There are a couple people I have in my feed which I would prefer to not have in there (some I don't even know). Not even blocking removes them from the feed. Could this problem be addressed ?

**Comment by jfullstackdev** [comment]
> Not even blocking removes them from the feed. Could this problem be addressed ?

that should be the feature of blocking them

hopefully some Staff will see this post, so they will be aware of this

**Comment by Nolram12345** [comment]
The option to block them is even linked in the context menu on the feed - but it doesn't accomplish anything at all. The user is blocked, but they still appear on the feed.
An option to remove users from your feed without blocking them would be great (and having them blocked remove them from your feed).

**Comment by TurnrDev** [comment]
This! I get a bot user @weblate fill my feed with spam. I don't want to block it because I use it for translations on my own repo, but really don't care about other repos it contributes to.

**Comment by chshersh** [comment]
I would like to be able to remove users from the feed of people to follow without blocking them.

I prefer not to block people on GitHub because blocking may have weird consequences (not having access to repositories or issues you've participated in before). I don't want to interact with some people and they don't want to interact with me. So I'd love to be able to remove people from my "Who to follow" recommendations.
